src/python/pants/backend/adhoc/run_system_binary.py

```diff
@@ -79,7 +81,15 @@ async def _find_binary(
 
         return binary
 
-    raise ValueError(f"Could not find a binary with `{binary_name}` with the provided constraints.")
+    raise ValueError(
+        f"Could not find a binary with `{binary_name}`"
```

Comment:
```suggestion
        f"Could not find binary `{binary_name}`"
```

Reply:
@chrisjrn doesn't this read weird to you?
> Could not find a binary with `bash`

Reply:
afaict the current version should say "a binary with _name_ `bash`"?

Reply:
that is indeed better. why doesn't GH show that change have been made here then... 🤔